### ISSUE TITLE
Address space inheritance for blob URLs

### ISSUE THREAD

**Issue by anforowicz** [WICG/private-network-access#18]
If a loopback or local frame creates and iframes a blob URL, then what should be the behavior of the subframe wrt cors-rfc1918?  I think all frames with origin X (e.g. https://intranet.corp.example.com) should behave consistently and therefore I expect the same cors-rfc1918 behavior for both 1) https://intranet.corp.example.com and 2) blob:https://intranet.corp.example.com/some-guid.

**Comment by letitz** [comment]
I tend to agree. What we care about is the provenance of the content, so the blob you describe should not behave differently from the page that created it.

**Comment by letitz** [comment]
I was going to post an update here, but I think this issue is actually simply superseded by #27.

TL;DR: I think we should inherit the address space of the blob URL creator context.

**Comment by letitz** [comment]
Re-opening this to track longer term work, blocked upstream by https://github.com/w3c/FileAPI/issues/142.

We want to implement the inheritance laid out in the previous comment, but cannot do so until the Blob URL store can capture policies at URL creation time.

**Comment by letitz** [comment]
This issue is fixed via integration with HTML's new [policy container](https://html.spec.whatwg.org/multipage/origin.html#policy-container), added in 4627c13084301ad962c4d562a1f9cf26c2c7cc39. Implementation work to match the spec should not be tracked here.
